### ISSUE TITLE
After copying a list, changing an item on the new list changes the same item in all lists.

### ISSUE THREAD

**Issue by DarkStarX1** [galenmaly/lighterpack#22]
First off.. this app is great. Thanks!

I've run into this issue several times. I've tried clearing the cache, logging back in, etc. For instance.. I have a three lists (High, Med, Low) and I change out the big three items (pack, tent, bag) on each to figure up different weight scenarios. The issue arises when I copy one of the lists to a new list, change one of existing items, and instead of just changing the item entry for that list only, it changes the item entry on all lists. So now, instead of my High list having the original 4LB pack, it now has the 1.5lb pack I added to the newly copied list.

TL;DR

1. Create a list and add an item.
2. Copy that list
3. Update the item in the new list.
4 The item is also changed on both lists.

**Comment by galenmaly** [comment]
This functionality is intended. Due to prior confusion about this feature in February I added a note on the "copy list" dialog explaining this behavior:

![copy-warn-before](https://cloud.githubusercontent.com/assets/7111499/14234633/6e60a892-f99c-11e5-9f58-61a9853c67d4.png)

I have just made an additional change to make this note more prominent:

![copy-warn-after](https://cloud.githubusercontent.com/assets/7111499/14234635/7908ee9e-f99c-11e5-9189-5dc11330702e.png)
